pkg/subctl/cmd/benchmark.go

```diff
@@ -36,16 +37,19 @@ var (
 )
 
 func init() {
-	msg := "Runs the test within a single cluster"
-	benchmarkLatencyCmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster", false, msg)
-
-	benchmarkThroughputCmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster", false, msg)
+	addBenchmarkFlags(benchmarkLatencyCmd)
+	addBenchmarkFlags(benchmarkThroughputCmd)
 
 	benchmarkCmd.AddCommand(benchmarkThroughputCmd)
 	benchmarkCmd.AddCommand(benchmarkLatencyCmd)
 	rootCmd.AddCommand(benchmarkCmd)
 }
 
+func addBenchmarkFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster", false, "Runs the test within a single cluster")
+	cmd.PersistentFlags().BoolVar(&verbose, "verbose", true, "Produce verbose logs during benchmark tests")
```

Comment:
We normally have verbose = false in other parts
```suggestion
	cmd.PersistentFlags().BoolVar(&verbose, "verbose", false, "Produce verbose logs during benchmark tests")
```

Reply:
Yes, see https://github.com/submariner-io/submariner-operator/issues/649 for general discussion.